docs/design/job_architecture.md

```diff
@@ -414,114 +246,287 @@ In case of error, the response has the keys:
 
 The most common log error encountered is that logs are not found -- this can occur if the job has not yet started running or the job was terminated whilst it was still in the job queue.
 
-**bus** `LOGS`
 
-Log data is split out into individual jobs and sent to `job_id` (see above)
+### `RETRY` (response)
 
+Response to a request to retry one or more jobs. If the request was successful, includes the new job state; otherwise, contains an error message.
 
-### `RETRY`
-Sent when one or more jobs are retried
+Retry data indexed by the ID of the job being retried. For each job retried, the following data is returned:
+  * `job_id` - string, ID of the job that was retried (the retry parent)
+  * `job` - object, the job state in the format used by the [`STATUS`](#status-response)
+  * `retry_id` - string, ID of the new job
+  * `retry` - string, the job state object of the new job that was launched in the format used by the [`STATUS`](#status-response)
+
+In case of error, the response has instead the keys:
+  * `job_id`
+  * `error` - brief message explaining the issue
 
-**content**
-Dictionary with key(s) original job ID and value dictionaries with the following structure:
+Sample response JSON:
 ```json
 {
   "job_id_1": {
     "job_id": "job_id_1",
-    "job": {"jobState": {"job_id": "job_id_1", "status": status, ...} ...},
+    "job": {"jobState": {"job_id": "job_id_1", "status": "error"}, "job_id": "job_id_1", "outputWidgetState": null},

```

Comment:
For some reason the renderer is complaining about these `null` entries, which doesn't make sense to me